src/components/message-box/_message_box.scss

```diff
@@ -0,0 +1,24 @@
+$includeHtml: false !default;
+
+@if ($includeHtml) {
+  .sg-message-box {
+    position: relative;
+    display: inline-block;
```

Comment:
Why `inline-block`?

Reply:
yeah 👍

Reply:
This is for it not to fill entire available space, this can be achieved by passing `fullWidth` prop